toolkit/geoview-compose/src/main/java/com/arcgismaps/toolkit/geoviewcompose/MapView.kt

```diff
@@ -523,6 +523,13 @@ public inline fun rememberLocationDisplay(
     }
 }
 
+/**
+ * The receiver class of the [MapView] content lambda.
+ *
+ * @since 200.5.0
+ */
+public class MapViewScope(_mapView: MapView?) : GeoViewScope(_mapView, "MapView")
```

Comment:
Constructor must be internal, parameter should not start with `_`.
Why is the parameter nullable?
```suggestion
public class MapViewScope internal constructor(mapView: MapView) : GeoViewScope(mapView, "MapView")
```

Reply:
This is just some oversight on my part, from trying to port over the changes in the inheritance hierarchy from the existing implementation. I have updated it.